lib/galaxy/webapps/galaxy/api/history_contents.py

```diff
@@ -673,6 +673,29 @@ def update(self, trans, history_id, id, payload, **kwd):
         else:
             return self.__handle_unknown_contents_type(trans, contents_type)
 
+    @expose_api_anonymous
+    def validate(self, trans, history_id, history_content_id, payload=None, **kwd):
+        """
+        update( self, trans, history_id, id, payload, **kwd )
+        * PUT /api/histories/{history_id}/contents/{id}/validate
+            updates the values for the history content item with the given ``id``
+
+        :type   history_id: str
+        :param  history_id: encoded id string of the items's History
+        :type   id:         str
+        :param  id:         the encoded id of the history item to validate
+
+        :rtype:     dict
+        :returns:   TODO
```

Comment:
Should that be fixed? Should this return an error object?

Reply:
It is asynchronous - because it launches an external set metadata job effectively - so we won't have that yet.

Reply:
I guess ``PUT`` s should just return empty bodies and use status codes to indicate what happened.

Reply:
Ok, maybe I misread the update_batch() function.